src/version.h

```diff
@@ -11,14 +11,14 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70926;
+static const int PROTOCOL_VERSION = 70927;
 
 //! initial proto version, to be increased after version/verack negotiation
 static const int INIT_PROTO_VERSION = 209;
 
 //! disconnect from peers older than this proto version
 static const int MIN_PEER_PROTO_VERSION_BEFORE_ENFORCEMENT = 70923;
```

Comment:
need to set `MIN_PEER_PROTO_VERSION_BEFORE_ENFORCEMENT` to `70926` here as we don't want to revert to letting ancient versions connect.

Reply:
Got it squashed the commit also